modules/message-lane/src/lib.rs

```diff
@@ -75,13 +75,17 @@ decl_storage! {
 }
 
 decl_event!(
-	pub enum Event {
+	pub enum Event<T, I=DefaultInstance> where
```

Comment:
```suggestion
	pub enum Event<T, I = DefaultInstance> where
```

Reply:
Weird that it isn't covered by `cargo fmt`.

Reply:
I think this is because we're inside a macro, so it doesn't really make sense to format code since you could break the code the macro is expecting